save-backend/src/main/kotlin/org/cqfn/save/backend/service/ProjectService.kt

```diff
@@ -0,0 +1,13 @@
+package org.cqfn.save.backend.service
+
+import org.cqfn.save.backend.repository.ProjectRepository
+import org.cqfn.save.entities.Project
+import org.springframework.stereotype.Service
+
+@Service
+class ProjectService (private val projectRepository: ProjectRepository) {
+
+    fun addResults(project: Project) {
```

Comment:
What is this method doing? Saving project to projects table when asked to save results?)